desktop/apps/artwork/components/commercial/test/template.coffee

```diff
@@ -39,4 +39,16 @@ describe 'Commercial template', ->
     $ = cheerio.load(html)
     $('input[name=email]').length.should.eql 1
 
+  it 'correctly displays on loan without pricing info', ->
+    html = renderArtwork { availability: 'on loan', sale_message: 'On loan' }
+    $ = cheerio.load(html)
+    $('.artwork-commercial__sale-message').text().should.eql 'On loan'
+    $('.artwork-commercial__shipping-info').length.should.eql 0
```

Comment:
If `On loan` and `Permanent Collection` have pricing info (which is technically possible in gravity) even though it will not be enabled in CMS - we should not show it either. Right?

Should here be a test for that as well?

Reply:
Sure 👍